src/equations/compressible_euler_1d.jl

```diff
@@ -232,14 +232,14 @@ are available in the paper:
     if v_normal <= 0.0
         sound_speed = sqrt(equations.gamma * p_local / rho_local) # local sound speed
         p_star = p_local *
-                 (1 + 0.5 * (equations.gamma - 1) * v_normal / sound_speed)^(2 *
+                 (1 + 0.5f0 * (equations.gamma - 1) * v_normal / sound_speed)^(2 *
                                                                              equations.gamma *
                                                                              equations.inv_gamma_minus_one)
```

Comment:
```suggestion
                 (1 + 0.5f0 * (equations.gamma - 1) * v_normal / sound_speed)^(2 *
                                                                               equations.gamma *
                                                                               equations.inv_gamma_minus_one)
```
That's something that should be fixed by running the formatter